traffic_portal/app/src/common/api/DeliveryServiceService.js

```diff
@@ -17,168 +17,184 @@
-            .then(
-                function() {
-                    messageModel.setMessages([ { level: 'success', text: 'Steering target created' } ], true);
-                    locationUtils.navigateToPath('/delivery-services/' + dsId + '/targets');
-                },
-                function(fault) {
-                    messageModel.setMessages(fault.data.alerts, false);
-                }
-            );
+        return $http.post(ENV.api['root'] + 'steering/' + dsId + '/targets', target).then(
+            function(result) {
+                messageModel.setMessages(result.data.alerts, true);
+                locationUtils.navigateToPath('/delivery-services/' + dsId + '/targets');
+            },
+            function(err) {
+                messageModel.setMessages(err.data.alerts, false);
+            }
+        );
     };
 
     this.deleteDeliveryServiceTarget = function(dsId, targetId) {
-        return Restangular.one('steering', dsId).one('targets', targetId).remove()
-            .then(
-                function() {
-                    messageModel.setMessages([ { level: 'success', text: 'Steering target deleted' } ], true);
-                    locationUtils.navigateToPath('/delivery-services/' + dsId + '/targets');
-                },
-                function(fault) {
-                    messageModel.setMessages(fault.data.alerts, true);
-                }
-            );
+        return $http.delete(ENV.api['root'] + 'steering/', dsId + '/targets/' + targetId).then(
+            function(result) {
+                messageModel.setMessages(result.data.alerts, true);
+                locationUtils.navigateToPath('/delivery-services/' + dsId + '/targets');
+            },
+            function(err) {
+                messageModel.setMessages(err.data.alerts, true);
+            }
+        );
     };
 
     this.getUserDeliveryServices = function(userId) {
-        return Restangular.one('users', userId).getList('deliveryservices');
+        return $http.get(ENV.api['root'] + 'users/' + userId + '/deliveryservices').then(
+            function(result) {
+                return result.data.response;
+            },
+            function(err) {
+                console.error(err);
+            }
+        );
     };
 
     this.deleteDeliveryServiceServer = function(dsId, serverId) {
-        return httpService.delete(ENV.api['root'] + 'deliveryservice_server/' + dsId + '/' + serverId)
-            .then(
-                function() {
-                    messageModel.setMessages([ { level: 'success', text: 'Delivery service and server were unlinked.' } ], false);
-                },
-                function(fault) {
-                    messageModel.setMessages(fault.data.alerts, true);
-                }
-            );
+        return httpService.delete(ENV.api['root'] + 'deliveryservice_server/' + dsId + '/' + serverId).then(
+            function(result) {
```

Comment:
shared-libs.js?built=1558641564542:66530 TypeError: Cannot read property 'data' of undefined
    at DeliveryServiceService.js:163